httpserver/http_connection.cpp

```diff
@@ -409,7 +412,7 @@ void connection_t::process_swarm_req(boost::string_view target) {
         const json body = json::parse(request_.body(), nullptr, false);
 
         if (body == nlohmann::detail::value_t::discarded) {
-            LOKI_LOG(error, "Bad snode test request: invalid json");
+            LOKI_LOG(debug, "Bad snode test request: invalid json");
```

Comment:
Can we add a body_stream_ message here like below for consistency?

Reply:
Also a couple times below

Reply:
updated